tools/debugCatcher.js

```diff
@@ -0,0 +1,15 @@
+const Helper = require('@codeceptjs/helper');
+
+class DebugCatcher {
+  constructor() {
+    this.messages = '';
+    const orig = Helper.prototype.debug;
```

Comment:
why are working with prototype? this is not recommended - debug function is normally accessible by this.debug()

Reply:
Discussed - this is OK.